crates/bonsaidb-macros/tests/collection.rs

```diff
@@ -103,3 +97,37 @@ fn serialization_none() {
         bonsaidb::core::schema::CollectionName::new("Authority", "Name")
     );
 }
+
+// TODO actually test the functionality
+// This could be done through using the macro in the tests for encryption
+#[test]
+// Pretty pointless, maybe also error?
```

Comment:
yeah, but `:101` might also best be resolved through using it for test  at different places? not sure how to test if encryption works :D

Reply:
I was going to add a separate comment then deleted it because I wasn't quite sure if that's what you meant by functionality. I will be adopting these derives throughout the test suites and examples (collection is already done, going to do views today), so the functionality will be tested for sure.